spec/forms/policy_form_spec.rb

```diff
@@ -1,16 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe PolicyForm do
+  include PublishingApiContentHelpers
+
+  before do
+    stub_content_calls_from_publishing_api
+    stub_any_publishing_api_write
+    stub_any_publishing_api_publish
+    stub_post_to_search
+  end
+
+  let(:attributes) do
```

Comment:
given we have a couple of different sets of attributes kicking about in this spec, might be a touch clearer if we call this `policy_attributes`.